.chromatic/custom-addons/chromatic/index.js

```diff
@@ -15,12 +15,18 @@ export const withChromaticProvider = makeDecorator({
     } else {
       selectedLocales = options.locales ? locales.map(l => l.value).slice(1) : ['en-US', 'ar-AE'];
     }
-    let height = options.height;
+    let height;
+    let minHeight;
+    if(isNaN(options.height)) {
+      minHeight = 1000;
```

Comment:
What stories are broken by not having a height?

Reply:
If we don't set this, then dialogs get cut off because the button alone doesn't take much height, so the dialog ends up partially off the screen